common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1087,6 +1087,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "DATETIME_FIELD_OUT_OF_BOUNDS" : {
+    "message" : [
+      "The value <badValue> you entered for <unit> is not valid. Please provide a value between <range>. To disable strict validation, you can turn off ANSI mode by setting <ansiConfig> to \"false\"."
```

Comment:
Could you please follow the same way of providing ANSI turn off suggestion, as it makes it easier to track what error messages are related to ANSI. `If necessary set <ansiConfig> to "false" to bypass this error.` Removal of this message is related to https://issues.apache.org/jira/browse/SPARK-49642